week_01/code_wars/myeong/200828_GetMiddle.cpp

```diff
@@ -0,0 +1,15 @@
+std::string get_middle(std::string input) 
+{
+  int mid;
+  mid=input.length()/2;
+  
+  if ( input.length()==1 ) {
+    return input;
+  }
+  else if ( input.length()%2==1 && input.length()!=1 ) {
```

Comment:
```c++
if ( input.length()==1 ) {
    return input;
  }
...
```

위의 if 문에서 이미 input 길이가 1인 것을 체크하기 때문에, `input.length()!=1` 이 조건은 필요 없을 것 같네~

```suggestion
  else if ( input.length()%2==1) {
```